docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,101 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [pages section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#pages).
 
+## Price Lists
+The following price lists webhook events fire in response to actions that affect a specific price list:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/pricelist/created`  | Fires when a price list is created. | 
+| `store/pricelist/updated/` | Fires when a price list is updated. | 
+| `store/pricelist/activated/` | Fires when a price list is activated. | 
+| `store/pricelist/deactivated/` | Fires when a price list is deactivated. | 
+| `store/pricelist/deleted/` | Fires when a price list is deleted. | 
+| `store/pricelists/deleted/` | Fires when price lists are deleted. | 
```

Comment:
Would it make sense to add a sample payload below for this hook, as it's a batch webhook?

Reply:
The batch webhook has the same payload.